e2e/testdata/e2e/pod-volume-device-template.yaml

```diff
@@ -7,8 +7,8 @@ metadata:
 spec:
   containers:
     - name: ubuntu
-      image: quay.io/cybozu/ubuntu:20.04
-      command: ["/usr/local/bin/pause"]
+      image: ubuntu:20.04
+      command: ["/usr/bin/sleep", "infinity"]
```

Comment:
can we use pause?

Reply:
e2e test cases use some commands in this container.

https://github.com/topolvm/topolvm/blob/main/e2e/e2e_test.go